Source/Scene/Model.js

```diff
@@ -45,6 +45,7 @@ define([
         './getModelAccessor',
         './ModelAnimationCache',
         './ModelAnimationCollection',
+        './modelMaterialsCommon',
```

Comment:
@pjcozzi It's just a function. It looks like all things that are just a function are camel case. Is that not correct?